### ISSUE TITLE
Home page accessibility issues

### ISSUE THREAD

**Issue by sveltifier** [ONSvisual/local-statistics#21]
1. Skip to main content link not working.
2. closed "more content" sections are visuallyhidden. Should be display:none.
3. Breadcrumb should be outside of <main> tag ideally.
4. Some colours don't invert on high contrast/inverted mode.
5. Autocomplete might not be marked up as a combobox?
6. Postcode search result needs to be announced. Ideally should be on another page.
7. Postodes might be better being a description list <dl> than a <table>, and worth considering making it a modal.
